multiversx_sdk/core/transaction_computer.py

```diff
@@ -122,6 +126,16 @@ def _to_dictionary(self, transaction: ITransaction) -> Dict[str, Any]:
         if transaction.guardian:
             dictionary["guardian"] = transaction.guardian
 
+        if transaction.relayer:
+            dictionary["relayer"] = transaction.relayer
+
+        if len(transaction.inner_transactions):
+            inner_transactions = []
+            for inner_transaction in transaction.inner_transactions:
+                inner_transactions.append(self._to_dictionary(inner_transaction))
+
+            dictionary["innerTransactions"] = [self._to_dictionary(transaction=tx, with_signature=True) for tx in transaction.inner_transactions]
```

Comment:
i think this can simply be:
```py
if len(transaction.inner_transactions):
    dictionary["innerTransactions"] = [self._to_dictionary(transaction=tx, with_signature=True) for tx in transaction.inner_transactions]
```

Reply:
you are right :+1: